Readme.md

```diff
@@ -0,0 +1,10 @@
+# Regarding flatpak-pip-generator
+
+I failed to get it working with the default script, something always went wrong,
+so I tweaked it to prefer the platform-independent .whl files over other archives,
```

Comment:
There's a PR for this in flatpak-builder-tools repo that you can use.

Reply:
Are you referring to [pip: Add support for pyproject.toml](https://github.com/flatpak/flatpak-builder-tools/pull/332)? Or the one that was already merged [Allow ignoring package in flatpak-pip-generator](https://github.com/flatpak/flatpak-builder-tools/pull/364)? (Or something else entirely?)

The former wouldn't exactly do anything for me, so I suppose you mean your own PR, the latter. It's interesting, but ultimately I could solve the issue better by removing the problematic dependencies from the requirements.txt, then adding back the dependencies of those removed dependencies that didn't cause issues.
The rest I could handle manually. The problem here in particular is that on a system with CUDA support (like mine, but I guess the buildbot server may or may not be affected?) it will always go for the cuda supporting versions, which are several GB larger. Took a while to stamp out that nonsense.

I tried the same using your new `--ignore-pkg` but the dependencies of the problematic ones still started pulling in cuda stuff. So well, the manual setup I did works, at least. Thanks for adding that feature for other edge cases though!

Reply:
No https://github.com/flatpak/flatpak-builder-tools/pull/331

Reply:
Ooooh, yeah that does fix that problem, thanks! That makes a lot more sense. Switching to it once it's merged should achieve the same effect then, probably in a less hacky way.